uaclient/tests/test_cli_status.py

```diff
@@ -607,26 +678,7 @@ def test_attached_formats(
                 {"name": "UA_FEATURES_ALLOW_BETA", "value": "true"},
             ]
 
-        if use_all:
-            services = SERVICES_JSON_ALL
-        else:
-            services = [
-                svc
-                for svc in SERVICES_JSON_ALL
-                if svc["name"] not in BETA_SVC_NAMES
-            ]
-
-        inapplicable_services = [
-            service["name"]
-            for service in RESPONSE_AVAILABLE_SERVICES
-            if not service["available"]
-        ]
-
-        filtered_services = [
-            service
-            for service in services
-            if service["name"] not in inapplicable_services
-        ]
+        services = SERVICES_JSON_ALL if use_all else SERVICES_JSON
```

Comment:
Testing locally, I have livepatch enabled, and this test fails because I get the message that Livepatch blocks FIPS in the json/yaml output... so it is actually calling status, which imho is a leak. Is there a sensible way to mock the status call to make sure this test is not dependent on the actual system?